packages/kotti-ui/source/kotti-row/index.ts

```diff
@@ -43,10 +44,12 @@ export const KtRow = attachMeta(
 						{
 							class: [
 								'kt-row',
-								props.justify !== 'start'
```

Comment:
This being caught implies that we didn’t properly unionify the `Justify` `enum` in `props`

Reply:
KtRow uses non-zodded legacy types. Skip this until is refactored to use make-props.